plumbing/api.go

```diff
@@ -194,6 +194,16 @@ func (api *API) ChainSyncHandleNewTipSet(ctx context.Context, ci *types.ChainInf
 	return api.syncer.HandleNewTipSet(ctx, ci, trusted)
 }
 
+// ChainExport exports the chain store to `out`.
+func (api *API) ChainExport(ctx context.Context, head types.TipSetKey, out io.Writer) (types.TipSetKey, error) {
+	return api.chain.ChainExport(ctx, head, out)
+}
+
+// ChainImport imports the chain with the same genesis block from `in`.
```

Comment:
I don't understand the "with the same genesis block" comment. Is that old?

Reply:
Old sorry. Import would fail when syncing the chain if the genesis block was different, since Import no longer syncs the imported chain this comment can be removed. Although (somewhat obviously) it still holds true that an imported chain with a different genesis block will fail to sync.